packages/admin/src/App.tsx

```diff
@@ -14,8 +14,9 @@ function App() {
       <Global styles={getAdminStyle()} />
       <BrowserRouter>
         <Switch>
-          <Route path="/login" component={Login} />
-          <Route exact path="/" component={Home} />
+          <Route exact path="/login" component={Login} />
+          <Route path="/scraper" component={ScraperPage} />
+          <Route path="/" component={ScraperPage} />
```

Comment:
<Route exact path={['/', '/login']} component={LoginContainer} />
```suggestion
          <Route path={["/","/scraper"]} component={ScraperPage} />
```

Reply:
그리고 컴포넌트에 Page를 붙이려면 공통적으로 Login도 LoginPage로 네이밍을 하는게 좋을 것 같아요
근데 어차피 route는 page단위여서..둘 다 안하는게 좋은것 같기도 하네요

Reply:
route는 page단위여서 상관없지만 `ScraperPage`라고 붙였으니 통일감 있게 Login도 `LoginPage`라고 붙이는 것도 좋은 것 같아요!

Reply:
LoginPage로 변경했어요~